src/linear_algebra/std_basis.lean

```diff
@@ -145,6 +152,26 @@ funext $ λ i, (finsupp.single_eq_pi_single i a).symm
 
 end linear_map
 
+namespace basis
+
+variables {R M n : Type*}
+variables [decidable_eq n] [fintype n]
+variables [semiring R] [add_comm_monoid M] [module R M]
+
+@[simp] lemma equiv_fun_symm_std_basis (b : basis n R M) (i : n) :
```

Comment:
I think this can be factored through `b.repr.symm` applied to `finsupp.basis`, that would be a useful lemma for the infinite-dimensional case.

Reply:
I am not sure whether this is what you had in mind, applying `b.repr` to the equation and and using `simp` gives a very short proof. This gives no new interesting lemmas.

Reply:
I've added this lemma, is this the one you wanted?
```
lemma test (a : R) (i : n) :
  finset.univ.sum (λ (x : n), finsupp.single x (ite (i = x) a 0)) = finsupp.single i a
```
Should probably named `finset_sum_single_ite`, right?